### PR TITLE
Fix YahooFinanceCSVData _loadline to support volume==Null

### PR DESCRIPTION
**Pull request by cgi1**

### CODE REVIEW

**Comment by cgi1** [comment]
Only one two if statements to provent zero devision (error comes from bf001239f91202150352646e7c978a52a114e06f) to support adjusted prices.

**Comment by cgi1** [comment]
Community reference: https://community.backtrader.com/post/2682

File: backtrader/feeds/yahoo.py [hunk 1]
```diff
@@ -108,15 +108,18 @@ def _loadline(self, linetokens):
...
+            else:
+              v = None
             if self.p.round:
                 decimals = self.p.decimals
                 o = round(o, decimals)
                 h = round(h, decimals)
                 l = round(l, decimals)
                 c = round(c, decimals)
-                v = round(v, decimals)
+                if v:
```
Comment (mementum) [inline backtrader/feeds/yahoo.py]: If v is `None`, then the operation with `round` would raise a `TypeError` exception (in both Python `2.7` and `3.x`)

**Comment by mementum** [comment]
There will be a check for volume `0` in the next release

**Comment by mementum** [comment]
The pull-request seems also to be against a 2 month old commit. There is nothing like the code above at line 111. And that code was for the old Yahoo format. The newest format executes a multiplication and not a division

**Pull request closed without merging**